### PR TITLE
Incorporate ephem plugin

### PR DESCRIPTION
**Pull request by domi4484**

### CODE REVIEW

**Comment by domi4484** [comment]
@3nids what should I do to get the new project appear on transifex? should i start the push workflow by hand or better wait for the merge?

**Comment by 3nids** [comment]
yes this will happen on merge only

File: share/python/plugins/kadas_ephem/resources.py
Comment (3nids) [inline share/python/plugins/kadas_ephem/resources.py:1]: Ideally this should be built on not in the source code, but it's the same for the moment for the other plugins.
@m-kuhn any opinion here?

**Review by 3nids — APPROVED** [review]

File: share/python/plugins/kadas_ephem/resources.py
Comment (domi4484) [reply share/python/plugins/kadas_ephem/resources.py:1]: we could as well just load the images directly. Or is there some advantage having it in a resource file for a plugin?

**Comment by 3nids** [comment]
@domi4484 I am getting an error in milx

ModuleNotFoundError: No module named 'ephem' 
Traceback (most recent call last):
  File "C:\LegacySW\kadas-milx-portable\kadas-win64-2024-09-27_0545\kadas-win64-2024-09-27_0545\bin\Lib\site-packages\qgis\utils.py", line 426, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\LegacySW/kadas-milx-portable/kadas-win64-2024-09-27_0545/kadas-win64-2024-09-27_0545/share/kadas/python/plugins\kadas_ephem\__init__.py", line 15, in classFactory
    from .ephem_plugin import EphemPlugin
  File "C:\LegacySW\kadas-milx-portable\kadas-win64-2024-09-27_0545\kadas-win64-2024-09-27_0545\bin\Lib\site-packages\qgis\utils.py", line 966, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\LegacySW/kadas-milx-portable/kadas-win64-2024-09-27_0545/kadas-win64-2024-09-27_0545/share/kadas/python/plugins\kadas_ephem\ephem_plugin.py", line 19, in 
    from .ephem_tool import EphemTool
  File "C:\LegacySW\kadas-milx-portable\kadas-win64-2024-09-27_0545\kadas-win64-2024-09-27_0545\bin\Lib\site-packages\qgis\utils.py", line 966, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\LegacySW/kadas-milx-portable/kadas-win64-2024-09-27_0545/kadas-win64-2024-09-27_0545/share/kadas/python/plugins\kadas_ephem\ephem_tool.py", line 2, in 
    import ephem
  File "C:\LegacySW\kadas-milx-portable\kadas-win64-2024-09-27_0545\kadas-win64-2024-09-27_0545\bin\Lib\site-packages\qgis\utils.py", line 966, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'ephem'

**Pull request merged**